src/main/java/net/sf/jabref/bibtex/BibtexSingleFieldProperties.java

```diff
@@ -5,11 +5,11 @@
 public enum BibtexSingleFieldProperties {
```

Comment:
Yes, that is the new class for this. Now renamed to FieldProperties (it might be that the removal was not done, on my office machine it isn't enough to do git add . to include all changes...).